docs/source/configuration/logging.mdx

```diff
@@ -1,8 +1,29 @@
 ---
-title: Configuring Logging
-description: Tuning logging configurations
+title: Logging
+description: Logging output formats
 ---
 
 import { Link } from 'gatsby';
 
```

Comment:
Should we add a section about setting the verbosity of logging?

Reply:
Added